sdk/python/foundation-models/system/finetune/image-classification/multiclass-classification/hftransformers-fridgeobjects-multiclass-classification.ipynb

```diff
@@ -172,6 +172,10 @@
    "source": [
     "finetune_cluster_name = \"sample-finetune-cluster-gpu\"\n",
     "\n",
+    "# You can create a new cluster or use an existing cluster for model evaluation.\n",
```

Comment:
I would create a cell with model eval cluster creation. There I will use the same name as that of FT cluster.